src/renderer/components/WelcomeScreen/WelcomeScreen.tsx

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useMemo} from 'react';
+import {useIntl, FormattedMessage} from 'react-intl';
+import classNames from 'classnames';
+
+import bullseye from '../../assets/svg/bullseye.svg';
+import channels from '../../assets/svg/channels.svg';
+import chat2 from '../../assets/svg/chat2.svg';
+import clipboard from '../../assets/svg/clipboard.svg';
```

Comment:
eslint should have caught this, but we generally places relative imports in the end of the imports order 0/5

Reply:
Updated.